### PR TITLE
Azure Monitor Container Insights and recommended alerts for Azure Arc-enabled Kubernetes

### PR DESCRIPTION
**Pull request by lanicolas**
New scenario to integrate Azure Arc-enabled Kubernetes with Azure Monitor to enable Container Insights as well as enable all the recommended alerts by pabloameijeirascanay

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    }
+    ```
+
+    > **NOTE: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/azure/role-based-access-control/best-practices)**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation:
+
+- User has deployed a Kubernetes cluster and has it connected as Azure Arc-enabled Kubernetes cluster.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:82]:
```suggestion
- User has deployed a Kubernetes cluster and has it connected as an Azure Arc-enabled Kubernetes cluster.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+## Create Azure Monitor cluster extensions instance
+
+To create a new extension instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Azure Monitor cluster extension on your Azure Arc-enabled Kubernetes cluster.
+
+- Before integrating the cluster with Azure Monitor, click on the "Extensions" tab for the connected Azure Arc cluster to show how the cluster is not currently being assessed by Azure Monitor.
+
+    ![Screenshot showing Azure Portal with Azure Arc-enabled Kubernetes resource extensions](./01.png)
+
+- Navigate to the folder that has the kind cluster definition
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:100]: Do you mean the folder that has the script ?

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+  ```shell  
+  . ./azure_monitor_alerts.
+  ```
+
+    > **NOTE: The extra dot is due to the shell script having an _export_ function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+   The script will:
+
+  - Login to your Azure subscription using the SPN credentials
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:128]:
```suggestion
  - Login to your Azure subscription using the service principal credentials
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+  - Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
+  - Create Azure Monitor cluster extension instance
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:130]:
```suggestion
  - Create the Azure Monitor cluster extension instance
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+- You can also verify the pods by running the command below:
+
+  ```bash
+  kubectl get pod -n kube-system --kubeconfig <cluster-name>.kubeconfig | grep omsagent
+  ```
+
+  ![Screenshot extension pods on cluster](./04.png)
+
+- Verify under the Alert rules tab on alerts section of the Azure Arc-enabled Kubernetes cluster that the alert rules are correctly installed.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:145]:
```suggestion
- Verify under the Alert rules tab on alerts section of the Azure Arc-enabled Kubernetes cluster that the alert rules are correctly created.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+  ![Screenshot showing Azure Portal with Azure Arc-enabled Kubernetes resource alerts rules](./05.png)
+
+## Simulate an alert
+
+- To verify that the recommended alerts are working properly, create the below pod to simulate an OOMKilledContainers alert:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:151]: Better to show steps on how to apply this deployment to the cluster

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+  - _`tenantId`_ - Your Azure tenant ID
+  - _`resourceGroup`_ - Azure resource group name
+  - _`arcClusterName`_ - Azure Arc Cluster Name
+  - _`azureLocation`_ - Azure region
+  - _`logAnalyticsWorkspace`_ - Log Analytics Workspace Name
+  - _`k8sExtensionName`_ - Azure Monitor extension name, should be azuremonitor-containers
+  - _`actionGroupName`_ - Action Group for the Alerts
+  - _`email`_ - Email for the Action Group
+
+    ![Screenshot parameter examples](./02.png)
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:116]: Align resource group name with other scenarios "Arc-K8s-Demo"

**Review by sebassem — COMMENTED** [review]
Minor lingo
Resource group naming
Extra step to show deployment steps

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+In this scenario, you will hook the Azure Arc-enabled Kubernetes cluster to Azure Monitor Container Insights by deploying the [Azure Monitor cluster extension](https://docs.microsoft.com/azure/azure-monitor/containers/container-insights-enable-arc-enabled-clusters?toc=%2Fazure%2Fazure-arc%2Fkubernetes%2Ftoc.json#create-extension-instance-using-azure-cli) on your Kubernetes cluster in order to start collecting Kubernetes related logs and telemetry. Then the recommended alerts will be enabled by an [ARM template](https://docs.microsoft.com/es-es/azure/azure-resource-manager/templates/overview).
+
+  > **NOTE: This guide assumes you already deployed a Kubernetes cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion**
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/container_insights/_index.md:15]:
```suggestion
  > **NOTE: This scenario assumes you already deployed a Kubernetes cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion**
```

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**